src/modules/roles/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const createRoleRoute = require('./createRoleRoute')
+const getRoleRoute = require('./getRoleRoute')
+
+const router = express.Router();
+
+
+router.use('/create-role', createRoleRoute)
+router.use('/get-role', getRoleRoute)
+
+
+module.exports = router;
```

Comment:
I think we can do something like below to refactor this.

```suggestion
const express = require('express');
const router = express.Router();
const path = require('path');
const glob = require('glob');

glob.sync(path.join(__dirname, './*.js')).forEach((file) => {
    if (file !== __filename) {
        router.use(require(file));
    }
});

module.exports = router;

```